src/form-container/form-container.jsx

```diff
@@ -0,0 +1,27 @@
+import './form-container.css';
+import React from 'react';
+import SearchComponent from '../search-component/search-component';
+import CounterComponent from '../counter-component/counter-component';
+import ToggleComponent from '../toggle-component/toggle-component';
+
+class FromContainer extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { isSearchVisible: true };
+  }
+
+  onToggleChange = (newState) => this.setState({ isSearchVisible: newState });
+
+  render() {
+    const { isSearchVisible } = this.state;
+    return (
+      <div className="form-container">
```

Comment:
if it's form, shouldn't it have tag `form` and `onSubmit` event? but it doesn't fit with counter component, maybe to separate them? make form with search and counter component